pepagent/pepagent.py

```diff
@@ -302,50 +371,71 @@ def get_namespaces(
             if names_only:
                 return [n[0] for n in namespaces]
 
-        return [self.get_namespace(n) for n in namespaces]
+        namespaces_list = []
+        for ns in namespaces:
+            try:
+                namespaces_list.append(self.get_namespace(ns))
+            except TypeError:
+                _LOGGER.warning(
+                    f"Warning: Error in collecting projects from database. {ns} wasn't collected!"
+                )
 
-    def get_anno(
+        return namespaces_list
+
+    def get_project_annotation(
         self,
         registry: str = None,
```

Comment:
`registry_path` ?

I guess a registry is something like dockerhub, or pephub, which serves data, right?